conda/cli/conda_argparse.py

```diff
@@ -12,6 +12,7 @@
     _CountAction,
     _HelpAction,
 )
+from functools import chain
```

Comment:
I think chain is in ```itertools``` rather than ```functools```.

I'm not sure if this is something that has moved around between Python versions

https://docs.python.org/3/library/itertools.html